### DISCUSSION TITLE
Improve UX to make it very intuitive there are more choices (display hints, force select all, 1-click choice)

### DISCUSSION THREAD

**Discussion by Vadorequest** [lukevella/rallly#1175] [Ideas]
We've migrated from Doodle to Rallly for one main reason: Too many people were missing choices because they wouldn't see it (bad UX).

The same issue applies to Rallly, although it's "less worse" there. There are other discussion about that, such as:
- https://github.com/lukevella/rallly/discussions/143

> The main goal of Rallly is to make it easy for people to share their availability. If they are missing options/answers by not seeing those, it breaks the whole point of using it in the first place. So, this kind of issue should be top priority.

What could make this a non-issue would be to:
- Display how many choices were answered (e.g: 4/24) using different colors/sizes and icons (✅/❌) to make it obvious something is missing
  - Maybe replace the "click x3" by 3 very small buttons so that we can always give an answer for a date in a single click
- Add an option to force people to give an answer to every choice (efficient, would ensure all options are treated despite being a bit more annoying)
- Consider "not answered" as an answer by itself, and avoid replacing it by "not available", so that we may identify people who haven't answered for a date, and see with them if that was on purpose

Here is a video that explains the above in a more visual way:
https://www.loom.com/share/6e590284b26e4da8aaa1204496815b9a

**Comment by lukevella** [comment]
Thanks for taking the time to share your feedback. I want to take a step back and talk about tentative voting before discussing your points further. I believe the UX can be greatly improved by simplifying the voting to just yes/no answers. I opened another discussion for this here: #1179
